src/bp-core/bp-core-rest-api.php

```diff
@@ -10,34 +10,6 @@
 // Exit if accessed directly.
 defined( 'ABSPATH' ) || exit;
 
-/**
- * Is the BP REST plugin is active?
- *
- * @since 5.0.0
- *
- * @return bool True if the BP REST plugin is active. False otherwise.
- */
-function bp_rest_is_plugin_active() {
-	return (bool) has_action( 'bp_rest_api_init', 'bp_rest' );
-}
```

Comment:
There's no emergency to remove this imho: it could be useful if in the future we need to provide a way to load a previous version of the BP REST API.